user-guide/Advanced_Modules/DOM/tutorials/DOM_Remove_FieldDescriptor_Definition.md

```diff
@@ -0,0 +1,212 @@
+# Step 2: Soft delete one of the fields
+
+The provided code below accomplishes the following:
+
+- The *FieldDescriptor* we want to delete is retrieved from the *SectionDefinition*. 
+
+- The *IsSoftDeleted* property of the field descriptor is set to *true*. 
+
+- The *SectionDefinition* is updated.
+
+```C#
+using System;
+using System.Linq;
+using Skyline.DataMiner.Automation;
+using Skyline.DataMiner.Net.Apps.DataMinerObjectModel;
+using Skyline.DataMiner.Net.Messages.SLDataGateway;
+using Skyline.DataMiner.Net.Sections;
+
+namespace Tutorial
+{
+    ///<summary>
+    /// This script will set the "IsSoftDeleted" property of the "Phone number" FieldDescriptor of the SectionDefinition to true. 
+    ///</summary>
+    public class Script
+    {
+        public void Run(Engine engine)
+        {
+            var moduleName = "people_app";
+            var domHelper = new DomHelper(engine.SendSLNetMessages, moduleName);
+
+            var peopleSectionDefinition = domHelper.SectionDefinitions.Read(
+                SectionDefinitionExposers.Name.Equal("PeopleSectionDefinition")).FirstOrDefault() as CustomSectionDefinition;
+            if (peopleSectionDefinition == null)
+            {
+                engine.GenerateInformation("Did not find the 'PeopleSectionDefinition'");
+                return;
+            }
+
+            var fieldDescriptors = peopleSectionDefinition.GetAllFieldDescriptors().ToList();
+            var phoneNumberFieldDescriptor = fieldDescriptors?.FirstOrDefault(one => one.Name == "Phone number");
+
+            if (phoneNumberFieldDescriptor == null)
+            {
+                engine.GenerateInformation("Did not find the 'Phone number' fieldDescriptor");
+                return;
+            }
+
+            // Update the fieldDescriptor
+            phoneNumberFieldDescriptor.IsSoftDeleted = true;
+            domHelper.SectionDefinitions.Update(peopleSectionDefinition);
+        }
+    }
+}
+```
+
+After you have soft-deleted the field descriptor, you should no longer see it in your low-code app while creating a new instance.
+
+![Step2](~/user-guide/images/DOM_Remove_FieldDescriptor_Definition_Step2.png)
+
+# Step 3: Remove field from existing instances
+
+The provided code below accomplishes the following:
+
+- All instances using the *SectionDefinition* are retrieved.
+
+- The *FieldValues* for that *FieldDescriptor* are removed from the section in the instances. 
+
+- The instances are updated. 
+
+```C#
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Skyline.DataMiner.Automation;
+using Skyline.DataMiner.Net.Apps.DataMinerObjectModel;
+using Skyline.DataMiner.Net.Messages.SLDataGateway;
+using Skyline.DataMiner.Net.Sections;
+using Skyline.DataMiner.Net.Helper;
+
+namespace Tutorial
+{
+    ///<summary>
+    /// This script will get all instances using the SectionDefinition and delete the fields for the "Phone number" FieldDescriptor.
+    ///</summary>
+    public class Script
```

Comment:
Same here, we could put the variable parts at the top so it easy to re-use. Maybe also add information events every page so the user can see the script is not stuck.